src/messages/List/ListItem.tsx

```diff
@@ -132,6 +143,9 @@ const ListItem: FC<IListItemProps> = props => {
 			data-testid={isHeaderElement ? "header-image" : "list-item"}
 			id={id}
 		>
+			{!isHeaderElement && dividerBefore && (
+				<div className={mainclasses.divider} aria-hidden="true" role="presentation" />
```

Comment:
This is just an empty div with only some styling and no contents. aria-hidden or role presentation is not necessary. div has no semantic meaning and is already ignored by a screen reader.
```suggestion
				<div className={mainclasses.divider} />
```